### PR TITLE
Feature/product service

### PR DESCRIPTION
**Pull request by rowinskidamian**
please review my code

### CODE REVIEW

**Review by ewa-git — APPROVED** [review]
looks ok

File: fakturogen-comarch-connector/src/main/java/pl/fakturogen/comarch/connector/dto/ComarchProductDTO.java [hunk 1]
```diff
@@ -22,4 +22,12 @@
     private Double saleGrossPrice;
     private Double quantity;
     private Long id;
+
+    public String get$id() {
+        return $id;
```
Comment (KrzKos) [inline fakturogen-comarch-connector/src/main/java/pl/fakturogen/comarch/connector/dto/ComarchProductDTO.java:27]: Just to remind you, that we should change that $id field name to something without "$". There could be some issue in the future.

**Review by KrzKos — APPROVED** [review]
Nice

File: fakturogen-comarch-connector/src/main/java/pl/fakturogen/comarch/connector/dto/ComarchProductDTO.java (hunk 1 above)
Comment (rowinskidamian) [reply fakturogen-comarch-connector/src/main/java/pl/fakturogen/comarch/connector/dto/ComarchProductDTO.java]: Just moved this out, as we will not use it anywhere. Thanks for checking :).

**Pull request merged**